Conceptpower+Spring/src/main/webapp/WEB-INF/views/home.jsp

```diff
@@ -191,54 +191,192 @@ function prepareMergeConcept(conceptId) {
 function createWrapper(wrapperId) {
   window.location = '${pageContext.servletContext.contextPath}/auth/conceptlist/addconceptwrapper?wrapperId=' + wrapperId;
 }
+
+function getListOfReviewRequests(conceptId, operation){
+	$.ajax({
+        type: "GET",
+        url: "${pageContext.servletContext.contextPath}/auth/request/"+conceptId+"/all",
+        success: function(response){
+        	var reviewHistory='';
+        	reviewHistory += '<div id="review-history-'+conceptId+'">';
+        	$.each(response, function(index, element){
+        		var reviewNumber = index + 1;
+        		reviewHistory += '<div>';
+        		reviewHistory += '<div  class="reviewhistory">'
+        		+'<h4 data-toggle="tooltip" data-placement="right"><b>'+reviewNumber+'. '+element.request+'</b></h4></div>'
+        		reviewHistory += '<div  id="review-'+index+conceptId+'">'
+        		
+        		$.each(element.comments, function(iterator, commentElement){
+        			reviewHistory += '<div><h6 style="color:#19586B; display:inline-block;">'+commentElement.createdBy.toUpperCase()+': </h6><span>'+'  '+commentElement.comment+'</span></div>';
+        		});
+        		
+        		reviewHistory += '</div></div>';
+        	});
+        	
+        	reviewHistory += '</div>';
+        	if(operation === 'resolve'){
+        		$('#reviewHistoryForResolve').html(reviewHistory);
+        	}else{
+        		$('#reviewHistoryForReopen').html(reviewHistory);
+        	}
+        },
+        error: function(e){
+        	if(operation === 'resolve'){
+        		$('reopenError').text('Fetching Review History Failed')
+        	}else{
+        		$('resolveCommentError').text('Fetching Review History Failed')
+        	}
+        }
+    });
+}
+
 $(document).ready(function(){
-    $(".fa-exclamation-triangle").click(function() {
-		$("#fetchRequests").val($(this).data("request"));
+	
+    var reviewId;
```

Comment:
global variables are always a source for errors; set data attributes on the elements you're working with and use those instead.